pom.xml

```diff
@@ -1318,6 +1318,12 @@
                         <preparationGoals>clean verify -DskipTests</preparationGoals>
                     </configuration>
                 </plugin>
+
+                <plugin>
+                    <groupId>ca.vanzyl.provisio.maven.plugins</groupId>
+                    <artifactId>provisio-maven-plugin</artifactId>
+                    <version>1.0.8</version>
```

Comment:
Latest version is `1.0.9`, let's try that

Reply:
Changed to 1.0.9